lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+msgid "Stop print"
+msgstr "Stoppa utskriften"
+
+# MSG_STOPPED c=20
+#: messages.c:111
+msgid "STOPPED."
+msgstr "STOPPAD."
+
+# MSG_SUPPORT c=18
+#: ultralcd.cpp:6756
+msgid "Support"
+msgstr "Support"
+
+# MSG_SELFTEST_SWAPPED c=16
+#: ultralcd.cpp:8180
+msgid "Swapped"
+msgstr "Bytt"
+
+# MSG_SELECT_FILAMENT c=20
+#: ultralcd.cpp:4706
+msgid "Select filament:"
+msgstr "V\xe1lj filament:"
+
+# MSG_TEMP_CALIBRATION c=14
+#: messages.c:112
+msgid "Temp. cal."
+msgstr "Temp. kal."
+
+# MSG_SELECT_TEMP_MATCHES_MATERIAL c=20 r=4
+#: ultralcd.cpp:4847
+msgid "Select temperature which matches your material."
+msgstr "V\xe1lj temperatur som passar ditt material."
+
+# MSG_CALIBRATION_PINDA_MENU c=17
+#: ultralcd.cpp:5812
+msgid "Temp. calibration"
+msgstr "Temp. kalibrering"
+
+# MSG_TEMP_CAL_FAILED c=20 r=8
+#: ultralcd.cpp:3771
+msgid "Temperature calibration failed"
+msgstr "Temperaturkalibrering misslyckades"
+
+# MSG_TEMP_CALIBRATION_DONE c=20 r=12
+#: messages.c:113
+msgid "Temperature calibration is finished and active. Temp. calibration can be disabled in menu Settings->Temp. cal."
+msgstr "Temperaturkalibreringen \xe1r klar och aktiv. Temp. kalibrering kan inaktiveras i menyn Inst\xe1llningar->Temp. kal."
+
+# MSG_FS_VERIFIED c=20 r=3
+#: ultralcd.cpp:7411
+msgid "Sensor verified, remove the filament now."
+msgstr "Sensor verifierad, ta bort filamentet nu."
+
+# MSG_TEMPERATURE c=18
+#: ultralcd.cpp:5673
+msgid "Temperature"
+msgstr "Temperatur"
+
+# MSG_MENU_TEMPERATURES c=15
+#: ultralcd.cpp:2047
+msgid "Temperatures"
+msgstr "Temperaturer"
+
+# MSG_FOLLOW_Z_CALIBRATION_FLOW c=20 r=9
+#: messages.c:47
+msgid "There is still a need to make Z calibration. Please follow the manual, chapter First steps, section Calibration flow."
+msgstr "Det finns fortfarande ett behov av att g\xefra Z-kalibrering. V\xe1nligen f\xeflj manualen, kapitel F\xefrsta stegen, avsnitt Kalibreringsfl\xefde."
+
+# MSG_TOTAL_FILAMENT c=19
+#: ultralcd.cpp:2735
+msgid "Total filament"
+msgstr "Totalt filament"
+
+# MSG_TOTAL_PRINT_TIME c=19
+#: ultralcd.cpp:2736
+msgid "Total print time"
+msgstr "Total utskriftstid"
+
+# MSG_TUNE c=18
+#: ultralcd.cpp:6653
+msgid "Tune"
+msgstr "St\xe1ll in"
+
+# MSG_TOTAL_FAILURES c=20
+#: messages.c:101
+msgid "Total failures"
+msgstr "Totala misslyckanden"
+
+# MSG_TO_LOAD_FIL c=20
+#: ultralcd.cpp:2137
+msgid "to load filament"
+msgstr "att ladda filament"
+
+# MSG_TO_UNLOAD_FIL c=20
+#: ultralcd.cpp:2141
+msgid "to unload filament"
+msgstr "att ta bort filament"
+
+# MSG_UNLOAD_FILAMENT c=16
+#: messages.c:114
+msgid "Unload filament"
+msgstr "Ta bort filament"
+
+# MSG_UNLOADING_FILAMENT c=20
+#: messages.c:115
+msgid "Unloading filament"
+msgstr "Tar bort filament"
+
+# MSG_TOTAL c=6
+#: messages.c:100
+msgid "Total"
+msgstr "Total"
+
+# MSG_USED c=19
+#: ultralcd.cpp:5928
+msgid "Used during print"
+msgstr "Anv\xe1nds under utskr"
+
+# MSG_MENU_VOLTAGES c=15
+#: ultralcd.cpp:2050
+msgid "Voltages"
+msgstr "Sp\xe1nning"
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.